### PR TITLE
Two separate lists for must-haves & should-haves

### LINKED ISSUE #1: Must-haves & should-haves
I think it’s be great if we split this into two separate files (or sections).

The first should contain what we consider the must-haves to be a bot list. This is most of the current content.

A second section can then be created which we can put best practice suggestions in that are should-haves for bot lists.

### PR DESCRIPTION
**Pull request by MattIPv4**
This pull request intends to close #1 

Implement two separate lists, with the first containing what we consider to be bare-minimum must-have requirements for any bot list. The second list contains what would be considered best practices that are should-haves for lists but not absolute requirements.

### CODE REVIEW

**Comment by MattIPv4** [comment]
Feedback would be appreciated for any further changes before final review & merge.

**Review by Habchy — COMMENTED** [review]
i approve we should definitely split these up, but there seems to be complications now because of other changes

File: README.md [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
    - Glitch is acceptable if:
       - Uptime is immaculate
       - A custom domain name is used (whilst still following point `1`)
-3. Should not display all bots on the homepage.
-4. Must be fully functioning.
+3. be fully functioning.
    - No broken links/buttons.
-5. Must have consistent uptime.
+4. have consistent uptime.
    - Minor downtime for fixing a problem is fine
```
Comment (Andre601) [inline README.md]: Maybe change this to `- Minor downtime for maintenance or fixing issues is fine`

File: README.md [hunk 2]
```diff
@@ -1,23 +1,44 @@
...
 
 ---
 
+*Bot lists should meet these requirements to be a good bot list.*
+
+## Your bot list should:
+
+1. not display all bots on the homepage.
+2. not be hosted on Glitch.
```
Comment (Andre601) [inline README.md:26]: Doesn't this conflict with point 2 in the must-list?

File: README.md [hunk 3]
```diff
@@ -1,23 +1,44 @@
...
+3. have an API with good documentation.
+   - API should support `GET`ting bot information
+      - Uses a JSON body format for data
+   - API should support `POST`ing bot stats
+      - Server/guild count
+      - Shard support
+      - Uses an Authorization header
+      - Uses a JSON body format for data
```
Comment (Andre601) [inline README.md]: This is essentially the same as the one under the `API should support GETing bot information`

File: README.md [hunk 4]
```diff
@@ -1,23 +1,44 @@
...
+4. respond with correct status codes.
+   - All pages that load correctly should respond with 200
```
Comment (Andre601) [inline README.md]: Maybe change to 2xx? Since there are other 200 codes that would also make sense?

**Review by Andre601 — CHANGES_REQUESTED** [review]
There are some minor things that imo should have some changes

File: README.md (hunk 2 above)
Comment (MattIPv4) [reply README.md:26]: No, the must-have permits Glitch usage, just not using their subdomain. This says that lists shouldn't use Glitch at all.

**Review by Andre601 — APPROVED** [review]

**Pull request merged**